vip-block-data-api.php

```diff
@@ -17,18 +17,22 @@
 
 namespace WPCOMVIP\BlockDataApi;
 
-define( 'WPCOMVIP__BLOCK_DATA_API__PLUGIN_VERSION', '0.2.0' );
-define( 'WPCOMVIP__BLOCK_DATA_API__REST_ROUTE', 'vip-block-data-api/v1' );
+if ( ! defined( 'VIP_BLOCK_DATA_API_LOADED' ) ) {
```

Comment:
Should we throw a warning if it's already defined? Something is probably wrong if this happens.